docs/api/Auth0.Core.Exceptions.ApiError.html

```diff
@@ -164,6 +164,31 @@ <h5 class="propertyValue">Property Value</h5>
       </tr>
     </tbody>
   </table>
+  <a id="Auth0_Core_Exceptions_ApiError_ExtraData_" data-uid="Auth0.Core.Exceptions.ApiError.ExtraData*"></a>
+  <h4 id="Auth0_Core_Exceptions_ApiError_ExtraData" data-uid="Auth0.Core.Exceptions.ApiError.ExtraData">ExtraData</h4>
+  <div class="markdown level1 summary"><p>Additional key/values that might be returned by the error such as <code>mfa_required</code>.</p>
+</div>
+  <div class="markdown level1 conceptual"></div>
+  <h5 class="decalaration">Declaration</h5>
+  <div class="codewrapper">
+    <pre><code class="lang-csharp hljs">[JsonProperty(&quot;extraData&quot;)]
+public Dictionary&lt;string, string&gt; ExtraData { get; set; }</code></pre>
+  </div>
+  <h5 class="propertyValue">Property Value</h5>
+  <table class="table table-bordered table-striped table-condensed">
+    <thead>
+      <tr>
+        <th>Type</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><span class="xref">System.Collections.Generic.Dictionary</span>&lt;<span class="xref">System.String</span>, <span class="xref">System.String</span>&gt;</td>
+        <td></td>
+      </tr>
+    </tbody>
+  </table>
```

Comment:
Looks like docs wasnt run when adding this property 5 months ago https://github.com/auth0/auth0.net/commit/9f66bb347a18e13a87a56e18b7b2728488d4450b